authors/apps/authentication/tests/test_user_signup.py

```diff
@@ -0,0 +1,60 @@
+# Django and restframework imports
+from django.urls import reverse
+from rest_framework import status
+# Local imports
+from authors.apps.authentication.models import User
+from authors.apps.authentication.serializers import UserSerializer
+from .base import BaseTestCase
+
+
+class UserSignUpTestCase(BaseTestCase):
+    """
+    Class Test Case for testing user signup 
+    functionality
+    Class also tests the user model
+    """
+
+    def test_user_model(self):
```

Comment:
I do not think we should have this test at all because:
- It does not test any endpoints
- It does not affect the coverage
It tests that the user model has an email, username, and password which are the same fields that have to be tested by the signup test.